testsuite/FixedLocalTuple/TestFixedLocalTupleList.py

```diff
@@ -0,0 +1,127 @@
+#  Copyright (C) 2017
+#      Max Planck Institute for Polymer Research
+#  
+#  This file is part of ESPResSo++.
+#  
+#  ESPResSo++ is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#  
+#  ESPResSo++ is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#  
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+
+
+import espressopp
+from espressopp import unittest
+import mpi4py.MPI as MPI
+import math
+
+from espressopp import Real3D
+
+class TestFixedLocalTupleList(unittest.TestCase) :
+
+    def test0Lattice(self) :
+        system = espressopp.System()
+        
+        rng  = espressopp.esutil.RNG()
+        
+        N    = 4
+        SIZE = float(N)
+        box  = Real3D(SIZE)
+        bc   = espressopp.bc.OrthorhombicBC(None, box)
+        
+        system.bc = bc
+        
+        # a small skin avoids rounding problems
+        
+        system.skin = 0.001
+    
+        cutoff = SIZE/2. - system.skin 
+        
+        comm = espressopp.MPI.COMM_WORLD
+        
+        nodeGrid       = espressopp.tools.decomp.nodeGrid(espressopp.MPI.COMM_WORLD.size)
+        cellGrid       = espressopp.tools.decomp.cellGrid(box, nodeGrid, cutoff, system.skin)
+        
+        print 'NodeGrid = %s'%(nodeGrid,)
```

Comment:
you can simply omit tuple
`'NodeGrid = %s' % nodeGrid`
line in the bottom line